ajax/libs/vue-amap/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "vue-amap",
+  "version": "0.1.6",
+  "description": "A GaoDe Map Component Library for Vue.js.",
+  "filename": "app.js",
```

Comment:
It should point to the main minified file. In this case, it will point to `index.min.js`. Thanks.